CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 * Feature - Support pulling from Amazon ECR with specified IAM role in task definition
 * Feature - Enable support for task level CPU and memory constraints.
+* Bug - Fixed a bug where container state information wasn't reported. [#1067](https://github.com/aws/amazon-ecs-agent/pull/1067)
```

Comment:
Should all three of these bug fixes be in the `## 1.15.1` header for this merge as well?

Reply:
Yes, will update.

Reply:
Dev already has the other fixes and their corresponding changelog. The merge just pulls in the one entry that was missing.